example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java

```diff
@@ -16,21 +16,18 @@
  */
 package org.apache.rocketmq.example.ordermessage;
 
-import org.apache.rocketmq.client.exception.MQBrokerException;
-import org.apache.rocketmq.client.exception.MQClientException;
+
 import org.apache.rocketmq.client.producer.DefaultMQProducer;
 import org.apache.rocketmq.client.producer.MessageQueueSelector;
 import org.apache.rocketmq.client.producer.SendResult;
 import org.apache.rocketmq.common.message.Message;
 import org.apache.rocketmq.common.message.MessageQueue;
 import org.apache.rocketmq.remoting.common.RemotingHelper;
-import org.apache.rocketmq.remoting.exception.RemotingException;
 
-import java.io.UnsupportedEncodingException;
 import java.util.List;
 
 public class Producer {
-    public static void main(String[] args) throws UnsupportedEncodingException {
```

Comment:
Hello @alexcao2018 , Thanks for your contribution,
here is a tip,
Remove `UnsupportedEncodingException` before need to `RemotingHelper. DEFAULT_CHARSET` replace `StandardCharsets. UTF_8`, Because there is a compilation exception

Reply:
I have fixed what you mentioned in code.